src/style.css

```diff
@@ -1 +1,133 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+:root {
+  --font-family: Avenir, sans-serif;
+  --font-size-base: 16px;
+  --font-size-small: 14px;
+  --color-text: #3d4e61;
+  --color-border: #e1e7ed;
+  --box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  --box-shadow-hover: 0 3px 8px rgba(61, 78, 97, 0.2);
+  --gradient-focus: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+html {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  box-sizing: border-box;
+  font-family: var(--font-family);
+  font-size: var(--font-size-base);
+}
+
+.search-form {
+  padding: 0;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  margin: 20px auto 0;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Here you have a bottom margin of `0` and a top margin of `20px`. It's better to choose one direction for margins to avoid potential margin collapse issues.